src/vulnpy/django/vulnerable.py

```diff
@@ -1,6 +1,9 @@
 from django.http import HttpResponse
 
-from django.conf.urls import url as compat_url
+try:
+    from django.urls import re_path as compat_url
```

Comment:
you should be able to get coverage up to 100% if you switch the order of the imports - so first try importing from `django.conf.urls`, then in the except block resort to `django.urls`

Reply:
You are a wizard ! :D Thanks.